tt_metal/impl/dispatch/device_command.hpp

```diff
@@ -41,10 +41,10 @@ class DeviceCommand {
     DeviceCommand();
 
     enum class TransferType : uint8_t {
-        RUNTIME_ARGS,
```

Comment:
I'm don't think that moving these enums around are actually required. But can't hurt I guess